### PR TITLE
Fixed mysql tests.

### PR DESCRIPTION
**Pull request by ghost**
Changed default db backend to mysql + fixed broken tests.

### CODE REVIEW

**Comment by mkurek** [comment]
could you setup some CI service (circleci, travis) to run automatically for every PR? maybe we could start with https://github.com/allegro/django-powerdns-dnssec/pull/66 (circleci passed here)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+flake: 
+	flake8 --exclude migrations powerdns/
```
Comment (mkurek) [inline Makefile]: don't we manage powerdns too? models etc?

**Comment by szok** [comment]
:ok:

File: dnsaas/settings.py [hunk 2]
```diff
@@ -14,17 +14,18 @@
...
-        'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': ':memory:',
-            'USER': '',
-            'PASSWORD': '',
-            'HOST': '',
-            'PORT': '',
-        }
+DATABASES = {
+    'default': {
```
Comment (mkurek) [inline dnsaas/settings.py:18]: use 'TEST' entry to make sure that tests are executed in different db: https://github.com/allegro/ralph/blob/ng/src/ralph/settings/base.py#L114  (https://docs.djangoproject.com/ja/1.9/ref/settings/#test)

File: dnsaas/settings.py [hunk 3]
```diff
@@ -14,17 +14,18 @@
...
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'dnsaas',
+        'USER': os.environ.get('DNSAAS_MYSQL_USER', 'dnsaas'),
```
Comment (mkurek) [inline dnsaas/settings.py]: consider changing names to `DNSAAS_DB_XXX` to make it db-backend-agnostic.

File: Makefile (hunk 1 above)
Comment (ghost) [inline Makefile]: ?

Comment (mkurek) [inline Makefile]: my bad. powerdns it the directory to run flake for (first i thought it is excluded too). So another question - don't we want to check flakes in dnsaas too? :)

Comment (ghost) [inline Makefile]: Mostly it's empty files, like:
https://github.com/allegro/django-powerdns-dnssec/blob/develop/dnsaas/admin.py
https://github.com/allegro/django-powerdns-dnssec/blob/develop/dnsaas/tests.py
which dont pass flake8
I can fix flake8 (which results having above files empty) or rm those files from repo.

Comment (mkurek) [inline Makefile]: rm it

File: dnsaas/settings.py (hunk 2 above)
Comment (ghost) [inline dnsaas/settings.py:18]: done

**Comment by mkurek** [comment]
:ok:

**Pull request merged**